.teamcity/common/BuildCache.kt

```diff
@@ -22,8 +22,7 @@ interface BuildCache {
 
 data class RemoteBuildCache(val url: String, val username: String = "%gradle.cache.remote.username%", val password: String = "%gradle.cache.remote.password%") : BuildCache {
     override fun gradleParameters(os: Os): List<String> {
-        return listOf("--build-cache",
```

Comment:
I think we should still enable the build cache, given that the new plugin won't be used on `release`. There is also no downside to always run with `--build-cache` on CI, right?

Reply:
I cherry-picked the plugin yesterday to `release`. After this PR merged I'll also cherry-pick this to `release`.